### PR TITLE
[SSL] Cipher suites pt2

### PR DESCRIPTION
**Pull request by kodster28**
Closes out PCX-4675

### CODE REVIEW

File: content/ssl/reference/cipher-suites/_index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Cipher suites
+pcx_content_type: navigation
+weight: 1
+---
+
+# Cipher suites
```
Comment (kodster28) [inline content/ssl/reference/cipher-suites/_index.md:7]: This is just breaking content out from a really long page onto a separate one.

File: content/ssl/reference/cipher-suites/disable-cipher-suites.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Disable cipher suites
+pcx_content_type: how-to
+weight: 2
+meta:
+  title: Disable cipher suites — Edge certificates
+---
+
```
Comment (kodster28) [inline content/ssl/reference/cipher-suites/disable-cipher-suites.md]: This is just breaking content out from a really long page onto a separate one.

File: content/ssl/reference/cipher-suites/matching-on-origin.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: Match on origin
+pcx_content_type: how-to
+weight: 8
+meta:
+  title: Match on origin — Edge certificates
+---
+
```
Comment (kodster28) [inline content/ssl/reference/cipher-suites/matching-on-origin.md:8]: This is just breaking content out from a really long page onto a separate one.

File: content/ssl/reference/cipher-suites/recommendations.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+title: Recommendations
+pcx_content_type: configuration
+layout: list
+weight: 1
+meta:
+  title: Cipher suites recommendations — Edge certificates
```
Comment (kodster28) [inline content/ssl/reference/cipher-suites/recommendations.md:7]: This is just breaking content out from a really long page onto a separate one.

File: content/ssl/reference/cipher-suites/supported-cipher-suites.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+---
+title: Supported cipher suites
+pcx_content_type: reference
+layout: list
+weight: 1
+meta:
+  title: Supported cipher suites — Edge certificates
+---
+
+# Supported cipher suites — Edge certificates
```
Comment (kodster28) [inline content/ssl/reference/cipher-suites/supported-cipher-suites.md:10]: This is just breaking content out from a really long page onto a separate one.

File: content/ssl/reference/cipher-suites/compliance-status.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+pcx_content_type: reference
+layout: list
+weight: 6
+meta:
+  title: Compliance status — Edge cipher suites
+---
+
+# Compliance status
+
+When you choose various [cipher suites](/ssl/reference/cipher-suites/supported-cipher-suites) to use with your edge certificates, your application may need to be in compliance with various regulatory standards.
```
Comment (marciocloudflare) [inline content/ssl/reference/cipher-suites/compliance-status.md]:
```suggestion
When you choose various [cipher suites](/ssl/reference/cipher-suites/supported-cipher-suites/) to use with your edge certificates, your application may need to be in compliance with various regulatory standards.
```

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**